presentation/presentation.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2018 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// Package presentation prints results of an expression evaluation in
+// json and tabular formats.
+package presentation
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"strconv"
+	"time"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/open-policy-agent/opa/profiler"
+	"github.com/open-policy-agent/opa/rego"
+)
+
+// EvalResult holds the results from evaluation, profiling and metrics.
+type EvalResult struct {
+	Result      rego.ResultSet         `json:"result,omitempty"`
+	Explanation []string               `json:"explanation,omitempty"`
+	Metrics     map[string]interface{} `json:"metrics,omitempty"`
+	Profile     []profiler.ExprStats   `json:"profile,omitempty"`
+}
+
+// PrintJSON returns indented json output.
+func PrintJSON(x interface{}) (string, error) {
```

Comment:
Is there a reason this doesn't take a writer like the other PrintXXX functions?

Reply:
Updated method to accept writer